data/vaults/250/0x1DF3f4985Ac2103B96ed75f9d76F33696D2b1d66.json

```diff
@@ -6,7 +6,7 @@
   "withdrawalsDisabled": false,
   "order": 0,
   "migrationAvailable": false,
-  "apyOverride": 0.9329,
+  "apyTypeOverride": "NEW✨",
```

Comment:
I think it's just `new` since it's the type. NEW✨ should be got from the front end I think